web/src/store/index.ts

```diff
@@ -109,6 +111,13 @@ async function init(_router: VueRouter, loadUser = true, loginState = '' as stri
       state.userLoading = false;
     }
   }
+  try {
+    const appSettings = await api.getAppSettings();
+    state.bannerTitle = appSettings.portal_banner_title;
+    state.bannerMessage = appSettings.portal_banner_message;
+  } catch (exception) {
+    console.error(exception);
```

Comment:
I'd be OK with this particular ESLint rule being disabled for this line; like this:

```suggestion
    // eslint-disable-next-line no-console
    console.error(exception);
```